packages/react/src/components/header/components/navigationLink/navigationLinkDropdown/NavigationLinkDropdown.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { cloneElement, isValidElement, useRef, useState } from 'react';
+
+// import core base styles
+import 'hds-core';
+import styles from './NavigationLinkDropdown.module.scss';
+import { IconAngleDown } from '../../../../../icons';
+import classNames from '../../../../../utils/classNames';
+
+export enum NavigationLinkInteraction {
+  Hover = 'hover',
+  Click = 'click',
+}
+export enum DropdownMenuPosition {
+  Left = 'left',
+  Right = 'right',
+}
```

Comment:
Would it make sense to move these to ../types.ts as well?

Reply:
I thought of it but didn't want to do it because I thought it might confuse the users if they go import the `enum DropdownDirection` for prop usage and see these as importable enums as well, when they aren't actually usable to them. These are used under the hood by our code. What do you think, is that a valid concern? Or is there an easy way to have these there but not exported to the users?

Reply:
That's a good point, I think it's best to keep them here :)